src/hydra/hydraClient.js

```diff
@@ -11,43 +11,89 @@ import isPlainObject from 'lodash.isplainobject';
 import qs from 'qs';
 import fetchHydra from './fetchHydra';
 
+class AORDocument {
+  constructor(obj) {
+    Object.assign(this, obj, {
+      originId: obj.id,
+      id: obj['@id'],
+    });
+  }
+
+  /**
+   * @return {string}
+   */
+  toString() {
+    return `[object ${this.id}]`;
```

Comment:
we could also json.stringify, not sure how this is used.

Reply:
It’s just a hint when debugging